Fix error in request.js

talvarez0507 opened the pull request:
I incorrectly handled some .then() statements and only now realized that the code I merged has a bug. This PR fixes that bug.

talvarez0507 committed:
fix incorrect return statement

talvarez0507 committed:
fix parameter spacing

talvarez0507 committed:
remove incorrect parameter

jamesnam-google commented on portfolio/src/main/webapp/request.js:
```diff
@@ -55,7 +55,8 @@ function onClickStartScheduling() {
   // the web form.
-  fetchScheduledTasksFromServlet(scheduleRequest).then((scheduledTaskArray) => {
+  fetchScheduledTasksFromServlet().then((scheduledTaskArray) => {
+    scheduledTasks = scheduledTaskArray;
```
I think I mentioned this before, but do we need this scheduledTasks variable? Can it diverge from the DOM elements? It might be safer/simpler to treat DOM elements as the source of truth.

jamesnam-google reviewed (approved):
Approving the change with a comment

rgcase commented:
> I incorrectly handled some .then() statements and only now realized that the code I merged has a bug. This PR fixes that bug.

Hey Tomas. It's not really clear to me why this was a bug and how your PR fixes it. Can you leave a little more information about the bug, either as a comment here or in a maybe in a buganizer bug with the details?

talvarez0507 committed:
scheduledTask data is now DOM based

Now duration, description and priority are now displayed on the DOM. collectAllScheduledTasks() has also been changed from a local array that stores everything into a function that parses the DOM.

talvarez0507 committed:
revert small local change

talvarez0507 committed:
remove unused line

talvarez0507 committed:
fix lint errors
